llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -542,6 +543,9 @@ PassBuilder::buildO1FunctionSimplificationPipeline(OptimizationLevel Level,
   // opportunities that creates).
   FPM.addPass(BDCEPass());
 
+  // Simplify bit-packed operations before cleaning up with instcombine.
+  FPM.addPass(PackedIntegerCombinePass());
```

Comment:
Touching this definitely doesn't belong in this patch, and definitely requires phase ordering tests